content/hardware/08.edu/solution-and-kits/alvik/tutorials/user-manual/user-manual.md

```diff
@@ -112,133 +135,1280 @@ Alvik has five different sensors, all connected to the STM32 and accessible thro
+
+**Key Features:**
+- Detects lines on the ground (or any patterns using colors or materials with different reflective properties)
+- Helps in navigating predefined paths
+- Consists of three phototransistors and five infrared LEDs
+
+**How it Works:**
+
+**Infrared LEDs**: Emit infrared light towards the ground.
+**Phototransistors**: Detect the reflected infrared light.
+- When over a dark line, the reflection is low, indicating the presence of a line.
+- When over a light surface, the reflection is high, indicating no line.
+
+![Follow the line sensor principles](assets/ch07-blacktape-color.png)
+
+By continuously reading the values from the phototransistors, the robot can determine its position relative to the line and adjust its movements to follow the path accurately.
+
+### Color Sensor
+
+The color sensor on the Arduino Alvik robot is used to detect and identify colors on surfaces that the robot encounters. It provides both raw color readouts and labeled color information that can be used for various applications such as line following, object detection, and more. The sensor used is the APDS 9660.
+
+**Key Features:**
+- Detects and identifies colors
+- Provides raw color readouts and labeled color information
+- Useful for line following, object detection, and other applications
+
+![Color sensor bottom view](assets/imageColor.png)
+
+### IMU
+
+The Arduino Alvik robot is equipped with an onboard IMU (Inertial Measurement Unit) that provides valuable information about the robot's motion and orientation. The IMU can measure acceleration, angular velocity, and orientation. The sensor used is the LSM6DSOX.
+
+**Key Features:**
+- Measures acceleration, angular velocity, and orientation
+- Provides roll, pitch, and yaw values
+- Useful for motion tracking and stabilization
 
-Alvik has two high-precision geared motors and two RGB LEDs. The test programs are located in the same folder as the [sensor examples](#sensors).
+![IMU axis reference](assets/imageIMU.png)
+
+## Actuators
 
 | **Actuator name**        | **Part name**           | **Test program name** |
 |--------------------------|-------------------------|-----------------------|
 | Geared motors w/ encoder | GM12-N20VA-08255-150-EN | wheels_positions.py   |
 | RGB LEDs                 | RGB LEDs                | leds_settings.py      |
 
-#### Connectors
+The RGB LEDs can be used for visual feedback and can be programmed to display various colors and patterns. The LEDs take in **boolean** values for each color meaning that combinations are possible by setting each color **on** or **off** insteadof controlling intensity.
+
+### Connectors
 
 The connectors are placed in the back of the robot, the pinout is shown in the following image:
 
 ![Connectors Pinout](assets/datasheet_connectors.png)
 
-## Coding Alvik
 
-![Alvik USB Connection](assets/connecting-final.gif)
+## What the Robot Includes

```

Comment:
I would rename this section something else, we are talking here about Out of the box experience, my perception about "what the robot includes" is more related to physical aspects